packages/amplify_authenticator/lib/src/widgets/form_fields/sign_up_form_field.dart

```diff
@@ -251,7 +251,7 @@ abstract class SignUpFormField<FieldValue> extends AuthenticatorFormField<
     Key? key,
     required String title,
     required CognitoUserAttributeKey attributeKey,
-    required String hintText,
```

Comment:
I updated this for consistency with Confirm Sign In. I don't think hint text needs to be required. It isn't required for TextFormField / InputDecoration, and it makes sense to try to match to APIs.

Reply:
What happens if the customer creates a custom form field and doesn't supply hint text?

Reply:
It just won't display any hint text then

![Screen Shot 2021-11-23 at 12 51 31 PM](https://user-images.githubusercontent.com/20613561/143077890-5a07d398-081f-43ba-88b7-c919a8e6c9fd.png)
y